content/md/first-steps/manage-multidimensional-data-in-a-table.md

```diff
@@ -0,0 +1,143 @@
+Of course, you cannot define a negative value, and moreover, you are limited to 100 characters by the system. In other words, this parameter lets you define a **maximum number of characters between 1 and 100.**
+
+### Number column type
+In the `Number` column type, you can specify the `Minimum value`, the `maximum value` and if you allow decimal values or not.
+
+### Yes/No (boolean) column type
+There is no validation parameters in this column type!
+
+
+
+## Scalability of a table attribute
+In order to ensure that the PIM will still be running, we have defined some limits.  
+- You can create up to **10 columns** in a single table.
+- You can create up to **200000 options** within a Select column.
+- You can have up to **100 rows** per table attribute in a product sheet.
+- You can create up to **50 table attributes** in your PIM.
+- You can have up to **8000 product values** in a product sheet (including the whole product values + the table attributes content)
+
+
+## History tab
+As for the other attribute types, you retrieve the `History` tab.  
+In this tab, we track the activity on your attribute such as:
+- any change in the attribute `properties` tab
+- any change in the `table structure` tab, except if you update the option's labels (within a `select` column).
+- any change in the `label translations` tab
+
+
+# Import/export your table data
+
+## Import/export the table structure
+
+### Import/export the columns
+You can import/export a table attribute the same way as the other attribute types, by using the `attribute import` or `attribute export` profiles. But, the structure of your table (i.e, the columns of your table) must be written in a JSON format, and inserted in the dedicated column of your import/export file, called `table_configuration`.
+
+:::tips
+We are aware that this JSON format is not ideal, but there is no other easy way to proceed as of today. That's why, if you need to make some changes via flat files, we advise you to export your attributes first, fix what you need to fix, and re-import it.
+:::
+
+:::info
+If you prefer to use the API, please note that the column information are part of the `Attribute` endpoints. To learn more about it, you can refer to our [API documentation](https://api.akeneo.com/concepts/catalog-structure.html#attribute).
+:::
+
+Don't worry, we've developed dedicated import/export profiles for the options of your `select` column types :wink:.
+
+### Import/export the options of your 'select' columns.
+Because it would have been a nightmare to mix up the columns configuration and the options of the `Select` columns in the same file, we've decided to create distinct import and export profiles for the options: the `table option import` and `table option export`, both in CSV or XLSX format. To know more about imports, please check out [the dedicated article](imports.html), and for the exports, everything you need to know is in [this article](exports.html).
```

Comment:
```suggestion
Because it would have been a nightmare to mix up the columns configuration and the options of the `Select` columns in the same file, we've decided to create distinct import and export profiles for the options: the `table option import` and `table option export`, both in CSV or XLSX format. To know more about imports, please check out [the dedicated article](imports.html), and for exports, everything you need to know is in [this article](exports.html).
```